wqflask/wqflask/correlation/pre_computes.py

```diff
@@ -156,3 +160,39 @@ def get_datasets_data(base_dataset, target_dataset_data):
         samples_fetched, base_traits_data)
 
     return (target_results, base_results)
+
+
+def get_file_size(file_path):
+    """params:file_path
+       return:size of file in MB
```

Comment:
*issue:* I'll raise this in tomorrow's stand-up or on e-mail; but we need to come up with a standard way of writing doc-strings.